### PR TITLE
Add info on multiplexed ONT runs to Terra sample table

### PR DESCRIPTION
**Pull request by kvg**

### CODE REVIEW

**Comment by kvg** [comment]
To be merged after kvg_terra_add_flowcell_id .

**Comment by SHuang-Broad** [comment]
@kvg it looks like the branch has a conflict with `main`. 
Can you resolve it first?

**Comment by kvg** [comment]
Resolved!

File: terra/scripts/update_nanopore_tables.py [hunk 1]
```diff
@@ -52,11 +48,11 @@ def load_summaries(gcs_buckets, project):
 
     ts = []
     for gcs_bucket in gcs_buckets:
-        blobs = storage_client.list_blobs(re.sub("^gs://", "", gcs_bucket))
```
Comment (SHuang-Broad) [inline terra/scripts/update_nanopore_tables.py:55]: curiosity Q: do we see a lot of these timeouts?

File: terra/scripts/update_nanopore_tables.py [hunk 2]
```diff
@@ -52,11 +48,11 @@ def load_summaries(gcs_buckets, project):
...
+        blobs = storage_client.list_blobs(re.sub("^gs://", "", gcs_bucket), timeout=120)
 
         for blob in blobs:
             if 'final_summary' in blob.name:
-                doc = blob.download_as_string()
+                doc = blob.download_as_string(timeout=120)
```
Comment (SHuang-Broad) [inline terra/scripts/update_nanopore_tables.py:55]: trivial (I can do it later in the overall refactoring): how about making this 120 a constant in this module?

**Review by SHuang-Broad — APPROVED** [review]
I have some thoughts about various refactoring and optimizations that can be done, but they are optimizations. Let's get this in first.

**Pull request merged**